admin_guide/pruning_resources.adoc

```diff
@@ -211,13 +226,16 @@ is younger than `<duration>` relative to the current time. (default *60m*)
 |Prune each image that exceeds the smallest xref:limits.adoc#image-limits[limit]
 defined in the same project. This flag cannot be combined with `--keep-tag-revisions`
 nor `--keep-younger-than`.
-|===
 
-{product-title} uses the following logic to determine which images and layers to
-prune:
+.^|`--registry-url`
+|The address to use when contacting the registry. The command will attempt to
+use a cluster-internal URL determined from managed images and image streams. In
+case it fails (the registry cannot be resolved or reached), an alternative route
+that works needs to be provided using this flag.
+|===
```

Comment:
s/an alternative route that works needs to be provided using this flag/use this flag to provide an alternate route.